app/src/main/res/layout/fragment_home.xml

```diff
@@ -30,18 +30,21 @@
         <com.google.android.material.appbar.AppBarLayout
             android:id="@+id/appBarLayout"
             android:layout_width="match_parent"
-            android:layout_height="wrap_content">
+            android:layout_height="wrap_content"
+            android:touchscreenBlocksFocus="false">
 
             <com.google.android.material.appbar.CollapsingToolbarLayout
                 android:id="@+id/collapsingToolbarLayout"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
+                android:touchscreenBlocksFocus="false"
```

Comment:
Same question, is it needed on the collapsing toolbar here? Because you usually don't specify it on these views. Where exactly should this android:touchscreenBlocksFocus be placed?